projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -69,8 +69,6 @@ const BarChartInternal: FC< BarChartProps > = ( {
 	showZeroValues = false,
 } ) => {
 	const horizontal = orientation === 'horizontal';
-	// Generate a unique chart ID to avoid pattern conflicts with multiple charts
-	const internalChartId = useId();
 	const chartId = useChartId( providedChartId );
```

Comment:
It's even worse for `chartId` than `internalChartId` to have a conflict with other charts - the data registered in shared context would be overridden, so I think it's fine to just use `chartId` instead.